src/v/cluster/scheduling/leader_balancer.cc

```diff
@@ -167,11 +203,11 @@ ss::future<ss::stop_iteration> leader_balancer::balance() {
     if (!_raft0->is_leader()) {
         vlog(clusterlog.debug, "Leadership balancer tick: not leader");
         if (!_timer.armed()) {
-            _timer.arm(_idle_timeout);
+            _timer.arm(_idle_timeout());
         }
         _need_controller_refresh = true;
         co_return ss::stop_iteration::yes;
-    } else if (_raft0->config().brokers().size() == 0) {
+    } else if (_raft0->config().brokers().size() == 1) {
```

Comment:
is this a separate commit that fixes a bug?

Reply:
Yes, I forgot to separate it out - fixed.